docs/ref/models/conditional-expressions.txt

```diff
@@ -236,3 +241,28 @@ On other databases, this is emulated using a ``CASE`` statement:
 
 The two SQL statements are functionally equivalent but the more explicit
 ``FILTER`` may perform better.
+
+Conditional filter
+------------------
+
+.. versionadded:: 2.1
+
+When a conditional expression returns a boolean value, it is possible to
+use the expression directly in a filter expression. This means it will not
+be returned as a column in the query, but will still be used to filter
+results::
+
+    >>> non_unique_account_type = Client.objects.exclude(pk=OuterRef('pk')).filter(
+    ...     account_type=OuterRef('account_type'))
+    >>> Client.objects.filter(~Exists(non_unique_account_type))
+
+This results in SQL that is functionally the same as:
+
+.. code-block:: sql
+
+    SELECT *
+    FROM clients c1
```

Comment:
Nitpick but Django model tables usually use the singular form like you did in your inner query. Also, AFAIK, the `Exists` expression doesn't reduce the selected columns to `1` so it more along

```sql
SELECT *
FROM client c1
WHERE NOT EXISTS(
    SELECT *
    FROM client c2
    WHERE c2.account_type = c1.account_type AND NOT c2.id = c1.id
)
```

Reply:
(There are a couple of places above that seem to use `clients`, but `client` is nicer).

I did note "functionally the same as", but maybe it should change.